lms/djangoapps/bulk_email/tasks.py

```diff
@@ -676,7 +676,8 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         return subtask_status, None
     finally:
         # Clean up at the end.
-        connection.close()
+        pass
```

Comment:
https://pymongo.readthedocs.io/en/4.0/migrate-to-pymongo4.html#mongoclient-cannot-execute-operations-after-close

Its a breaking changes.